inst/rmarkdown/templates/oup/skeleton/skeleton.Rmd

```diff
@@ -100,51 +106,118 @@ A bullet list:
+
+# Literature citations
+
+By default, citations are handled by `natbib` using a numeric citation format. To use name-date citations, sets `namedate: TRUE` in the YAML header.
+
+Here are two sample references: 
+
+* **author (year) example:** @horvath2018dna showed some really cool things. Only seems to work properly if `namedate: TRUE`. 
+* **(author year) example:** This is a well known result [@ji20123d]. 
+
+The bibliography will appear at the end of the document.
+
+Though not normally available in the OUP LaTeX format, [CSL style files](https://www.zotero.org/styles) can also be used with the Rmarkdown adaptation by setting in the YAML header `citation_package: "default"` and defining the `csl` element to be the path towards the style file.
+
+# Equations
 
-## Generate a figure.
+An equation without a label for cross-referencing:
+
+$$
+E=mc^2
+$$
+
+An inline equation: $y=ax+b$
+
+An equation with a label for cross-referencing:
+
+\begin{equation}\label{eq:eq1}
+\int^{r_2}_0 F(r,\varphi){\rm d}r\,{\rm d}\varphi = 1
+\end{equation}
+
+This equation can be referenced as follows: Eq. \ref{eq:eq1}
+
+# Inserting R figures
+
+The code below creates a figure. The code is included in the output because `echo=TRUE`.
 
 ```{r fig1, fig.cap="This is the first figure.",echo=TRUE}
-plot(1:10,main="Some data",xlab="Distance (cm)",ylab="Time (hours)")
+plot(1:10,main="Some data",xlab="Distance (cm)",
+     ylab="Time (hours)")
 ```
 
 You can reference this figure as follows: Fig. \ref{fig:fig1}.
 
-```{r fig2, fig.cap="This is the second figure.",echo=TRUE}
-plot(1:5,pch=19,main="Some data",xlab="Distance (cm)",ylab="Time (hours)")
+## Figures spanning two-columns
+
+Figures can span two columns be setting `fig.env="figure*"`.
+
+```{r fig2, fig.cap="This is a wide figure.",fig.env="figure*"}
+plot(1:5,pch=19,main="Some wide data",
+     xlab="Distance (cm)",ylab="Time (hours)")
 ```
 
 Reference to second figure: Fig. \ref{fig:fig2}
 
+# Tables
+
 ## Generate a table using `xtable`
 
-```{r tab1, results="asis", echo=TRUE}
+```{r xtabletable, results="asis", echo=TRUE}
 df = data.frame(ID=1:3,code=letters[1:3])
 
-# Creates tables that follow OUP guidelines using xtable
+# Creates tables that follow OUP guidelines 
+# using xtable
 library(xtable) 
-print(xtable(df,caption="This is the table caption",label="tab:tab1"),
-      comment=FALSE)
+print(xtable(df,caption="This is a xtable table.",
+             label="tab:tab1"),
+      comment=FALSE,caption.placement="top")
 ```
 
 You can reference this table as follows: Table \ref{tab:tab1}.
 
 ## Generate a table using `kable`
 
-```{r anothertable, echo=TRUE}
+```{r kabletable, echo=TRUE}
 df = data.frame(ID=1:3,code=letters[1:3])
 
 # kable can alse be used for creating tables
-knitr::kable(df,caption="This is the table caption",format="latex",
+knitr::kable(df,caption="This is a kable table.",
+             #format="latex",
```

Comment:
If you add the option, you should remove the comment here. 
Otherwise, don't add the option and show how to use a format argument.

`kable()` can output also Markdown table, that will be converted to LaTeX by Pandoc and this should work too.

Reply:
Removed the comment.